### PR TITLE
EVEREST-1142 | Add command for validating RBAC policy

### PR DESCRIPTION
**Pull request by mayankshah1607**
Examples:

```shell
❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-1-good.csv
2024-07-08T16:48:06.735+0530    DEBUG   Debug logging enabled
✓ Valid

❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-2-bad.csv
× Invalid
policy syntax error - invalid policy line 'this is a bad policy'

❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-3-bad.csv
× Invalid
policy syntax error - invalid policy term 'monitoring instances'

❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-4-bad.csv
× Invalid
policy syntax error - role 'notexists:role' does not exist

❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-5-bad.csv
× Invalid
policy syntax error - invalid policy rule size: expected 4, got 5, rule: [adminrole:role monitoring-instances * */* asdfasdf]

❯ ./bin/everestctl settings rbac validate --policy-file ./pkg/rbac/testdata/policy-6-bad.csv
× Invalid
policy syntax error - unknown resource name 'non-existent-resource'
```

TODO
- Add `everestctl settings rbac can` command

### CODE REVIEW

File: pkg/rbac/validate.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+			continue
+		}
+		if !slices.Contains(roles, roleName) {
+			return fmt.Errorf("role '%s' does not exist", roleName)
+		}
+	}
+	return nil
+}
+
+func validateTerms(terms []string) error {
```
Comment (oksana-grishchenko) [inline pkg/rbac/validate.go:82]: These three functions look like good candidates for unit tests. However, if you don't see any value in testing them separately it's fine to keep only the `TestValidatePolicy`, just wanted to bring it to consideration.

**Review by oksana-grishchenko — APPROVED** [review]
LGTM, great job!🚀

File: pkg/rbac/validate.go (hunk 1 above)
Comment (mayankshah1607) [reply pkg/rbac/validate.go:82]: They're indirectly already covered in `TestValidatePolicy`, but I think there's no harm in adding some basic tests for this functions too. I will add them, thanks :)

Comment (mayankshah1607) [reply pkg/rbac/validate.go]: Done

**Pull request merged**